src/components/RelocationBox/RelocationBox.jsx

```diff
@@ -0,0 +1,393 @@
+    <ChakraProvider>
+      <div
+        // Conditional classes for approved/pending changes/rejected boxes to determine background coloring
+        className={`${styles.box}
+        ${status === 'evaluated' && approved ? styles['box-approved'] : ''}
+        ${status === 'evaluated' && approved === false ? styles['box-rejected'] : ''}
+        ${status === 'pending changes' ? styles['box-pending'] : ''}`}
+      >
+        <Accordion allowToggle>
+          <AccordionItem>
+            <h3>
+              <AccordionButton className={styles['accordian-button']}>
+                {/* Relocation box icon */}
+                <div className={styles['picture-div']}>
+                  <img src={RelocateBoxIcon} alt=" " />
+                </div>
+                {/* Box Number & date */}
+                <div className={styles['title-div']}>
+                  <p className={styles.title}>
+                    <p className={styles['box-number']}>Box #{boxID}</p>
+                    {date}
+                  </p>
+                </div>
+                <div className={styles['arrow-button']}>
+                  <AccordionIcon />
+                </div>
+              </AccordionButton>
+            </h3>
+            {/* Box picture */}
+            <AccordionPanel pb={4} className={styles['accordion-panel']}>
+              <div className={styles['box-details']}>
+                {picture === null ? (
+                  <div className={styles['image-wrapper']}>
+                    <img className={styles['image-box']} src={ImageVector} alt="" />
+                  </div>
+                ) : (
+                  <img src={picture} alt="" className={styles['image-corners']} />
+                )}
+                {/* Box Name */}
+                <FormControl>
+                  <FormLabel htmlFor="name" marginTop="5%">
+                    Name
+                  </FormLabel>
+                  <Input
+                    isReadOnly={status !== 'pending changes'}
+                    id="name"
+                    // type="text"
+                    value={boxHolderNameState}
+                    onChange={e => setBoxHolderNameState(e.target.value)}
+                  />
+                  {/* Box Email */}
+                  <FormLabel htmlFor="email" marginTop="5%">
+                    Email
+                  </FormLabel>
+                  <Input
+                    isReadOnly={status !== 'pending changes'}
+                    id="email"
+                    type="text"
+                    value={boxHolderEmailState}
+                    onChange={e => setBoxHolderEmailState(e.target.value)}
+                  />
+                  {/* Box Zip Code */}
+                  <FormLabel htmlFor="zipCode" marginTop="5%">
+                    Zip Code
+                  </FormLabel>
+                  <Input
+                    isReadOnly={status !== 'pending changes'}
+                    id="zipCode"
+                    type="number"
```

Comment:
I'm not sure if this actually does any error checking for the type, but we do accept international zip codes and some of those may be alphanumeric, so it's safer to keep type as text here.